src/parser/monk/mistweaver/CHANGELOG.js

```diff
@@ -5,6 +5,11 @@ import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 
 export default [
+  {
+    date: new Date('2019-03-11'),
+    changes: <>Updated look and feel of Mistweaver Azerite traits to conform to new 3.0 style.</>,
```

Comment:
This can be just a string 
```suggestion
    changes: 'Updated look and feel of Mistweaver Azerite traits to conform to new 3.0 style.',
```